src/features/events_list/readme.md

```
@@ -99,3 +107,74 @@ It has the following actions:
 
 - `setBBoxFilterFromCurrentMapView`
 - `resetBboxFilter`
+
+# Feature config
```

Markdownlint (0.37.0)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)